packages/identity-x/browser/index.js

```diff
@@ -1,7 +1,21 @@
+import * as Sentry from '@sentry/browser';
+import * as Integrations from '@sentry/integrations';
+import Vue from '@base-cms/marko-web/browser/vue';
 import components from './components';
 import createClient from './create-client';
 
-export default (Browser, mountPoint) => {
+const isProd = window.location.hostname !== '0.0.0.0';
+const dsn = 'https://7d01088298a44760a0e91923b1f17933@sentry.as3.io/17';
+
+export default (Browser, { mountPoint, sentryDisabled, sentryDsn } = {}) => {
+  if (!sentryDisabled) {
```

Comment:
Double-negative var...

Bring sentry vars in as a an object, not prefixed... e.g. `sentry.enabled` `sentry.dsn`